caktus_primer/static/js/modal.js

```diff
@@ -31,11 +36,11 @@ export class Modal {
 
     handleClick() {
         this.dialogTarget.showModal()
-        document.body.style.overflow = 'hidden';
+        document.body.style.overflow = 'hidden'
     }
 
     closeModal() {
         this.dialogTarget.close()
-        document.body.style.overflow = '';
+        document.body.style.overflow = ''
```

Comment:
Hmm … instead of repeating the `.overflow = ''` bit, could we split this out into `closeModal` (the action to close the modal) and `onClose` (the thing that handles the close event that results from closing it)?